toplevel/coqinit.ml

```diff
@@ -100,9 +106,9 @@ let libs_init_load_path ~load_init =
                            has_ml = AddTopML }
     } ] @
 
-  (* then standard library and plugins *)
-  [build_stdlib_path ~load_init ~unix_path:(coqlib/"theories") ~coq_path ~with_ml:false;
-   build_stdlib_path ~load_init ~unix_path:(coqlib/"plugins")  ~coq_path ~with_ml:true ] @
+  (* then standard library *)
+  [build_stdlib_ml_path ~dir:(coqlib/"plugins")] @
+  [build_stdlib_path ~load_init ~unix_path:(coqlib/"theories") ~coq_path ~with_ml:false] @
```

Comment:
Why keep the `with_ml` option of the `build_stdlib_path` since it's not used anymore?

Reply:
Well seen thanks, fixed.